### ISSUE TITLE
Simplify species filter categories 

### ISSUE THREAD

**Issue by SK53** [NERC-CEH/irecord-app#171]
Recently using I-record in the field I wanted to toggle filters to vascular plants & butterflies. In practice one needs to select/unselect Flowering Plants, Clubmosses, Ferns, Horsetails, and Conifer (also possibly Gingko) to toggle a standard recording group on or off.

In the end I reset back to all taxa which is slower (and can result in some awkward typos).

**Comment by kazlauskis** [comment]
Thanks, I agree the filtering should be simplified to fewer groups.

**Comment by kazlauskis** [comment]
@DavidRoy do you know if we could simplify the current species group list for filtering?

```
"acarine"
"acorn worm"
"alga"
"amphibian"
"annelid"
"arrow worm"
"bacterium"
"beardworm"
"bird"
"bony fish"
"bryozoan"
"centipede"
"clubmoss"
"coelenterate"
"comb jelly"
"conifer"
"crustacean"
"diatom"
"echinoderm"
"entoproct"
"false scorpion"
"fern"
"flatworm"
"flower. plant"
"foraminiferan"
"fungoid"
"fungus"
"gastrotrich"
"ginkgo"
"hairworm"
"harvestman"
"hornwort"
"undetermined"
"horseshoe worm"
"horsetail"
"alderfly"
"beetle"
"booklouse"
"bristletail"
"butterfly"
"caddis fly"
"cockroach"
"dragonfly"
"earwig"
"flea"
"lacewing"
"louse"
"mayfly"
"moth"
"orthopteran"
"scorpion fly"
"silverfish"
"snakefly"
"stick insect"
"stonefly"
"stylops"
"thrips"
"true bug"
"true fly"
"jawless fish"
"lampshell"
"lancelet"
"lichen"
"liverwort"
"marine mammal"
"millipede"
"mollusc"
"moss"
"peanut worm"
"priapulid"
"protozoan"
"quillwort"
"reptile"
"ribbon worm"
"rotifer"
"roundworm"
"sea spider"
"slime mould"
"spider"
"sponge"
"spoon worm"
"springtail"
"stonewort"
"symphylan"
"mammal"
"tunicate"
"two-tailed bristletail"
"unassigned"
"waterbear"
"trematode"
"tapeworm"
"scorpion"
"hymenopteran"
"chromist"
"gnathostomulid"
"cycliophoran"
"proturan"
"loriciferan"
"mud dragon"
"mesozoan"
"mantis"
"pauropod"
"cartilagenous fish"
"monogenean"
"thorny-headed worm"
"cyanobacterium"
"web-spinner"
```

**Comment by DavidRoy** [comment]
yes, we should definitely simplify.  Good to get the view of
@kitenetter
@StephRorke - relates to https://github.com/StephRorke/General-BRC/issues/27

**Comment by sacrevert** [comment]
Thanks Martin. Given this, the species group filtering could presumably be replaced with a list of schemes (selecting a scheme then filtering all nested taxa)? I think this would be preferable to the current situation. I think it would be useful to add this to the 6.2.0 milestone if there is capacity (it seems like a pretty easy win if the indicia tables already exist)

**Comment by kitenetter** [comment]
This needs more thought. Being able to choose "BSBI" rather than "Flowering Plants, Clubmosses, Ferns, Horsetails, Conifer and Gingko" would be a step forward, but having to choose lots of separate insect schemes (e.g. for someone interested in beetles) would be a step back, and raises the question of how to deal with taxa that are not covered by a scheme at all.

So I wonder whether we would be better off setting up a translation table that takes the UKSI informal groups and groups some of them together, and re-names some. I don't think the UKSI informal group names have ever changed much if at all, so this should be a one-off exercise.

@kazlauskis would such a translation table work for you? Would you need to implement it directly in the app or would it be best added to the Indicia data structure?

The sort of thing I am thinking of would be along these lines (happy to complete the process if this approach will work - will also need to check my understanding of some groups, e.g. not sure if everything that has "worm" in its name really is a worm!):

![image](https://github.com/user-attachments/assets/5c79c6de-a08c-4d20-b81b-d63eae9943f9)

**Comment by kitenetter** [comment]
Arguably the simplified names in the above table are not simple enough, and we could go further: 

![image](https://github.com/user-attachments/assets/13b85147-b4ef-4fc1-8fbc-8eeaee99f511)

@sacrevert assuming we are thinking about app users, would you want them to be able to distinguish between vascular plants and bryophytes?

Similar questions will arise elsewhere, e.g. some may prefer "butterflies", "moths", "all other insects".

**Comment by kitenetter** [comment]
It will be more than 14 when the whole list is done, but once Karolis confirms I will complete the list and we can see what it looks like.

**Comment by DavidRoy** [comment]
Is it worth looking at what other systems do? https://observation.org/species/search/ is probably our closest comparison but iNat is another one that's relevant?

**Comment by sacrevert** [comment]
No, because they have "mosses and lichens", and so shouldn't be taken seriously 😂

**Comment by kitenetter** [comment]
@kazlauskis the proposed table to map the group names is here:

[simplified taxon group names.zip](https://github.com/user-attachments/files/17962328/simplified.taxon.group.names.zip)

It brings the list down to 26 different names - some of these are arguable and other interpretations could be made, but hopefully these will work. And some are quite long, in the interests of making it clearer to users what the group contains; if any are too long let me know.

We could divide them up with subheadings, and/or we could order them by 'popularity' (e.g. based on number of records submitted per group), let me know if you think that is worth doing.

- amphibians and reptiles
- birds
- fish
- fungi and slime moulds
- insects: ants, bees, sawflies, wasps
- insects: beetles
- insects: butterflies
- insects: dragonflies and damselflies
- insects: grasshoppers and allies
- insects: moths
- insects: other groups
- insects: riverflies
- insects: true bugs and hoppers
- insects: true flies
- invertebrates: centipedes, millipedes
- invertebrates: crustaceans
- invertebrates: molluscs
- invertebrates: other groups
- invertebrates: spiders, harvestmen and other arachnids
- invertebrates: worms
- lichens
- mammals
- other groups
- plants
- plants: algae
- plants: mosses and relatives

**Comment by kazlauskis** [comment]
@kitenetter thanks, this looks good. What if we keep 10 top-level groups and then allow expanding on the ones that have more granular options (e.g. plants)? So users will first see 10 options and if they select "Plants", it automatically covers all the plant sub-categories, if they select "Plants" and "Algae" that will only show algae from that group.


- amphibians and reptiles
- birds
- fish
- fungi and slime moulds
- insects
	- ants, bees, sawflies, wasps
	- beetles
  - butterflies
  - dragonflies and damselflies
  - grasshoppers and allies
  - moths
  - other groups
  - riverflies
  - true bugs and hoppers
  - true flies
- invertebrates
  - centipedes, millipedes
  - crustaceans
  - molluscs
  - other groups
  - spiders, harvestmen and other arachnids
  - worms
- lichens
- mammals
- other groups
- plants
  - plants
  - algae
  - mosses and relatives

**Comment by sacrevert** [comment]
Fine by me, although the nested plants category should be "vascular plants", so

- plants
  - algae
  - mosses and relatives
  - vascular plants

**Comment by kitenetter** [comment]
Thanks @kazlauskis that approach (with Oli's amendment) looks great

**Comment by johnvanbreda** [comment]
Just a thought - would "other groups" be better represented as "other insect groups" and "other invertebrate groups" so that all the names makes sense whether shown within the hierarchy or shown standalone?

**Comment by japonicus** [comment]
The 'insects' _v's_ 'invertebrates' division seems a bit odd and is possibly likely to attract comment from pedants (not just me ;) )
Maybe the categories under 'invertebrates' should be raised to the top level?

**Comment by kitenetter** [comment]
Well, we could use "Non-insect invertebrates" and I have no objection to doing that, but personally I think it is clear what we mean (and am happy to sympathise with any pedants who may complain to us about it!). I don't think that the non-insect invertebrates should come up to top-level - I'm not expecting these filters to be used very often.
